client.go

```diff
@@ -1089,6 +1090,18 @@ func (c *Client) SetJSONEscapeHTML(b bool) *Client {
 	return c
 }
 
+// SetResponseBodyLimit set a max body size limit on response, avoid reading too many data to memory.
+//
+// Client will return [ErrResponseBodyTooLarge] if uncompressed response body size if larger than limit.
+// Body size limit will not be enforced in following case:
+// - ResponseBodyLimit is 0, which is the default behavior.
+// - [Response.SetOutput] is called to save a response data to file.
+// - `DoNotParseResponse` is set for client or request.
+func (c *Client) SetResponseBodyLimit(v int64) *Client {
```

Comment:
@trim21 Please test the godoc display by running godoc locally, e.g.:  bullet points in the godoc.